source/solvers/heat_transfer.cc

```diff
@@ -44,19 +43,18 @@ void
 HeatTransfer<dim>::assemble_system(
   const Parameters::SimulationControl::TimeSteppingMethod time_stepping_method)
 {
-  const double density = simulation_parameters.physical_properties.density;
-  const double specific_heat =
-    simulation_parameters.physical_properties.specific_heat;
-  const double thermal_conductivity =
-    simulation_parameters.physical_properties.thermal_conductivity;
-
-  const double viscosity = simulation_parameters.physical_properties.viscosity;
+  // Gather physical properties in case of mono fluids simulations (to be
+  // modified in case of multiple fluids simulations)
+  auto &physical_properties = this->simulation_parameters.physical_properties;
 
-  const double dynamic_viscosity = viscosity * density;
+  double density              = physical_properties.density;
+  double specific_heat        = physical_properties.specific_heat;
+  double thermal_conductivity = physical_properties.thermal_conductivity;
+  double viscosity            = physical_properties.viscosity;
 
-  const double rho_cp = density * specific_heat;
-
-  const double alpha = thermal_conductivity / rho_cp;
+  double dynamic_viscosity = viscosity * density;
```

Comment:
Is this still coherent with the new physical properties? I feel this "Having two viscosity" definition is gonna be painful if it is used elsewhere. Maybe it would be simpler to revert to just having kinematic viscosity and density defined and let the dynamic viscosity be dynamically refined when needed.